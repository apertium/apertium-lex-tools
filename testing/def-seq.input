^free<adj><sg>/libre<adj><sg>/gratis<adj><sg>$ ^whatever<n>/gloop<n>$
^free<adj><sg>/libre<adj><sg>/gratis<adj><sg>$ ^bloop<n>/gloop<n>$
^free<adj><sg>/libre<adj><sg>/gratis<adj><sg>$ ^shoop<n>/gloop<n>$
