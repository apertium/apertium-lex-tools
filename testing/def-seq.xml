<lrx>
  <def-seqs>
    <def-seq n="bloop">
      <or>
        <match lemma="bloop"/>
        <match lemma="shoop"/>
      </or>
    </def-seq>
  </def-seqs>
  <rules>
    <rule>
	    <match lemma="free" tags="adj.*"><select lemma="gratis"/></match>
    </rule>
    <rule weight="3">
      <match lemma="free" tags="adj.*">
        <select lemma="libre"/>
      </match>
      <seq n="bloop"/>
    </rule>
  </rules>
</lrx>
