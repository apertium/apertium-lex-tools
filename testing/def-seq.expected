^free<adj><sg>/gratis<adj><sg>$ ^whatever<n>/gloop<n>$
^free<adj><sg>/libre<adj><sg>$ ^bloop<n>/gloop<n>$
^free<adj><sg>/libre<adj><sg>$ ^shoop<n>/gloop<n>$
